### PR TITLE
Modify the judgment that skin effect is enable

### PR DESCRIPTION
**Pull request by troublemaker52025**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [x] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by troublemaker52025** [comment]
@cocos-robot run test cases

File: EngineErrorMap.md [hunk 1]
```diff
@@ -3355,16 +3343,24 @@ Camera '%s' clear flag is skybox, but skybox is disabled,  may cause strange bac
...
\ No newline at end of file
+There is no reflection probe in the scene or no probe is near the current object. No reflection probe will take effect on this object. Please create a new reflection probe or move existing ones closer.
+
+### 16303
+
+Separable-SSS skin filter need float output, please open ENABLE_FLOAT_OUTPUT define...
+
+### 16304
+
+Separable-SSS skin filter need set standard model, please check the isGlobalStandardSkinObject option in the MeshRender component.
```
Comment (jk20012001) [inline EngineErrorMap.md]: Skin material may need more accurate calculations, please select a head model of standard size, check the isGlobalStandardSkinObject option in the MeshRender component.

File: EngineErrorMap.md [hunk 2]
```diff
@@ -3355,16 +3343,24 @@ Camera '%s' clear flag is skybox, but skybox is disabled,  may cause strange bac
...
 
 ### 16302
 
-There is no reflection probe in the scene or no probe is near the current object. No reflection probe will take effect on this object. Please create a new reflection probe or move existing ones closer.
\ No newline at end of file
+There is no reflection probe in the scene or no probe is near the current object. No reflection probe will take effect on this object. Please create a new reflection probe or move existing ones closer.
+
+### 16303
+
+Separable-SSS skin filter need float output, please open ENABLE_FLOAT_OUTPUT define...
```
Comment (jk20012001) [inline EngineErrorMap.md]: Skin material needs floating-point render target, please check ENABLE_FLOAT_OUTPUT define in Project Settings--Macro

**Review by jk20012001 — APPROVED** [review]

**Comment by shaoqiangcai** [comment]
@cocos-robot run test cases

**Pull request merged**